tests/makeotfexe_data/input/spec/8a-11.fea

```diff
@@ -0,0 +1,13 @@
+# When the replacement glyph of a single substitution is the same as
+# the target glyph of a multiple substitution, the former is encoded
+# as an alternate rather than a single substitution. That should happen
+# with A by A.sc and C by C.sc below.
```

Comment:
Is this a new behavior or an old one?

Reply:
This is a test of behavior in both the old feat.c and new FeatCtx.cpp (implemented in `aaltCreate()` in both cases).

Reply:
(The addition of this test to the spec directory wasn't meant to have any strong implication. It's not clear that 8.a requires it. `makeotfexe` apparently does, though, so it needed a test.) (Edit: moved this to right thread.)

Reply:
Good to know.